### PR TITLE
Create psychologists types

### PR DESCRIPTION
**Pull request by desoindx**

### CODE REVIEW

File: services/updatePsyFields.ts [hunk 1]
```diff
@@ -10,7 +12,7 @@ const addFrenchLanguageIfMissing = (languages) => {
   return languages;
 };
 
-const editablePsyFields = (psy) => ({
+export const editablePsyFields = (psy: Psychologist): EditablePsychologist => ({
```
Comment (sandrica89) [inline services/updatePsyFields.ts:15]: All the functions here are exported twice (here and at the end of the file)

File: test/controllers/loginController.spec.js [hunk 2]
```diff
@@ -183,7 +183,7 @@ describe('loginController', async () => {
           .returns(
             Promise.resolve({
               email,
-              state: 'en_construction',
+              state: 'enConstruction',
```
Comment (sandrica89) [inline test/controllers/loginController.spec.js:186]: Shouldn't still be `en_construction` here? It's the value and not the enum.

File: test/seed/02-psychologists.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import clean from '../helper/clean';
 
-import { DOSSIER_STATE } from '../../utils/dossierState';
-import { generateUuidFromString } from '../../utils/uuid';
+import { DossierState } from '../../types/DemarcheSimplifiee';
```
Comment (sandrica89) [inline test/seed/02-psychologists.ts:3]: We could add `types` dir in the ts config to be able to import `types/DemarcheSimplifiee` directly

File: test/test-dbPsychologists.js [hunk 4]
```diff
@@ -129,41 +130,14 @@ describe('DB Psychologists', () => {
...
-
   describe('getActivePsychologists', () => {
     it('should only return not archived and accepte psychologists with capitalized lastName', async () => {
       const activePsy = clean.getOnePsy();
       const archivedPsy = clean.getOnePsy('archived@psy.fr');
       archivedPsy.archived = true;
       archivedPsy.lastName = 'ArchivedPsy';
       const constructionPsy = clean.getOnePsy('construction@psy.fr');
-      constructionPsy.state = 'en_construction';
+      constructionPsy.state = 'enConstruction';
```
Comment (sandrica89) [inline test/test-dbPsychologists.js:140]: Either use `en_construction` or `DossierState.enConstruction` (same for another test below)

File: test/test-psyProfileController.js [hunk 5]
```diff
@@ -624,6 +624,7 @@ describe('psyProfileController', () => {
 
     it('should activate psy', async () => {
       const psy = clean.getOneInactivePsy();
+      console.log(dbPsychologists);
```
Comment (sandrica89) [inline test/test-psyProfileController.js:627]: To remove if added for debug purpose only

**Review by sandrica89 — CHANGES_REQUESTED** [review]

File: services/updatePsyFields.ts (hunk 1 above)
Comment (desoindx) [reply services/updatePsyFields.ts:15]: I think it's ok for now, we will need to review all of our export in this task => https://trello.com/c/NYIiP3Dk/154-have-propre-export

File: test/seed/02-psychologists.ts (hunk 3 above)
Comment (desoindx) [reply test/seed/02-psychologists.ts:3]: https://trello.com/c/fBF734Qm/155-create-alias-in-the-backend-just-like-in-the-frontend

**Review by sandrica89 — APPROVED** [review]

**Pull request merged**